backend/src/main/resources/liquibase/db-changelog-1.0.xml

```diff
@@ -688,7 +689,7 @@
                 <constraints primaryKey="true" nullable="false"/>
             </column>
             <column name="migration_name" type="varchar(255)"/>
-            <column name="status" type="tinyint(1)"/>
+            <column name="status" type="tinyint"/>
```

Comment:
have you verified that this change won't break on all the different databases we support? This kind of change has broken things for customers before.

Reply:
rather than change this existing migration, could you just create a new one for H2? And, make sure this one doesn't run for H2?

Reply:
Fixed

Reply:
Fixed